lib/vdsm/storage/lvm.py

```diff
@@ -1162,8 +1165,10 @@ def deactivateUnusedLVs(vgname, skiplvs=()):
     # List prepared images LVs if any
```

Comment:
Wouldn't doing something like this achieve the same thing if we use lv names instead of full paths?
```python
def deactivateUnusedLVs(vgname, skiplvs=KEEPACTIVE):
```
I mean, technically, skipping LVs is already part of the API.

Reply:
Humm, we have the same lv name in different vg, that's why we prefer using complete path
ie : /dev/vg1/lv1, /dev/vg2/lv1

Reply:
Certainly `skiplvs=KEEPACTIVE` was not the best example here, as it mixes different vgs.

But my point was that there is already a logic for skiplvs in place, so I think it would be nice to consolidate the list before we get to the `if lv.name in skiplvs:` condition, filtering all lvs that pertain to `vgname`.

Something like (not tested):
```python
skiplvs = skiplvs + tuple(lvm.split('/')[-1] for lvm in KEEPACTIVE if vgname == lvm.split('/')[-2])
```
In this solution maybe worth using different than the complete path to ease the parsing.

Or even convert the `KEEPACTIVE` list into a dictionary, where the VG is the key and the value is a list of lvs that we can directly append to `skiplvs` based on `vgname`.